addons/prometheus/0.31.x/prometheus-1.yaml

```diff
@@ -105,11 +105,6 @@ spec:
                 targetPort: 5601
                 interval: 30s
         prometheusSpec:
-          image:
-            tag: v2.9.2
```

Comment:
This is what prevented your change in mesosphere/charts#309 from taking effect I'm guessing? 

But rather than delete entirely, should we still pin to a much more recent stable image? Or is that already covered since we're pointing to the chart `8.3.3`?

Reply:
Yeah, this was causing prometheus-operator to deploy an older version of Prometheus that couldn't handle some of the updates to the chart.

The chart's default images have been updated as part of the chart upgrade, so using defaults means we're getting the new versions. I decided to remove the tag override rather than update it, since AFAIK we no longer need to pin a specific version, and using the defaults should be easier to maintain.